zigbeetradfri/integrationpluginzigbeetradfri.cpp

```diff
@@ -637,6 +714,67 @@ void IntegrationPluginZigbeeTradfri::initOnOffSwitch(ZigbeeNode *node, ZigbeeNod
     });
 }
 
+void IntegrationPluginZigbeeTradfri::initShortcutButton(ZigbeeNode *node, ZigbeeNodeEndpoint *endpoint)
+{
+    // Get the current configured bindings for this node
+    ZigbeeReply *reply = node->removeAllBindings();
+    connect(reply, &ZigbeeReply::finished, node, [=](){
+        if (reply->error() != ZigbeeReply::ErrorNoError) {
+            qCWarning(dcZigbeeTradfri()) << "Failed to remove all bindings for initialization of" << node;
+        } else {
+            qCDebug(dcZigbeeTradfri()) << "Removed all bindings successfully from" << node;
+        }
+
+        // Read battery, bind and configure attribute reporting for battery
+        if (!endpoint->hasInputCluster(ZigbeeClusterLibrary::ClusterIdPowerConfiguration)) {
+            qCWarning(dcZigbeeTradfri()) << "Failed to initialize the power configuration cluster because the cluster could not be found" << node << endpoint;
+            return;
+        }
+
+        qCDebug(dcZigbeeTradfri()) << "Read power configuration cluster attributes" << node;
+        ZigbeeClusterReply *readAttributeReply = endpoint->getInputCluster(ZigbeeClusterLibrary::ClusterIdPowerConfiguration)->readAttributes({ZigbeeClusterPowerConfiguration::AttributeBatteryPercentageRemaining});
+        connect(readAttributeReply, &ZigbeeClusterReply::finished, node, [=](){
+            if (readAttributeReply->error() != ZigbeeClusterReply::ErrorNoError) {
+                qCWarning(dcZigbeeTradfri()) << "Failed to read power cluster attributes" << readAttributeReply->error();
+            } else {
+                qCDebug(dcZigbeeTradfri()) << "Read power configuration cluster attributes finished successfully";
+            }
+
+            qCDebug(dcZigbeeTradfri()) << "Bind on/off configuration cluster to coordinator";
+            ZigbeeDeviceObjectReply * zdoReply = node->deviceObject()->requestBindGroupAddress(endpoint->endpointId(), ZigbeeClusterLibrary::ClusterIdOnOff, 0x0000);
+            connect(zdoReply, &ZigbeeDeviceObjectReply::finished, node, [=](){
```

Comment:
same, a simple `bindOnOffInputCluster(endpoint)` would do once it inherits ZigbeeIntegrationPlugin